www/docs/mp/index.php

```diff
@@ -335,11 +336,24 @@
 # XXX This is current behaviour, but should probably now just be any recent MP
 $data['has_expenses'] = isset($MEMBER->extra_info['expenses2004_col1']) || isset($MEMBER->extra_info['expenses2006_col1']) || isset($MEMBER->extra_info['expenses2007_col1']) || isset($MEMBER->extra_info['expenses2008_col1']);
```

Comment:
`has_expenses` is presumably out of date in that all new MPs it'll be false? As per comment, should probably be any MP with a leave date since 2004 sometime. That'll then simplify the new code as well I think.